### ISSUE TITLE
Mage registry key "solarium_search_result" already exists"

### ISSUE THREAD

**Issue by srmobile** [jeroenvermeulen/magento-solarium#84]
I received an error stating that the Mage registry key "solarium_search_result" already exists". I'll admit I did try to install this over year ago (unsuccessfully).

Can someone point me where I could find the duplicate and remove it?

For now I've disabled the check on Mage.php to get by but would rather resolve this properly.

Thank you!

**Comment by toonvd** [comment]
This usually occurs because you didnt clear cache or you tried to install with compiler on.

**Comment by devdesco-ceo** [comment]
Same thing here. Compiler is disabled, all caches reset!

**Comment by ftsf** [comment]
also experiencing this with compiler disabled and caches cleared, although it is intermittent, some requests go through successfully, others get this error.

**Comment by toonvd** [comment]
@ftsf is this on a first install? Have you tried destroying your session?

**Comment by ftsf** [comment]
@toonvd yes, first install. It no longer seems to be occurring (it's a new day with a new session). Is this a known issue with existing sessions?

**Comment by toonvd** [comment]
It is possible since registry is session bound yes.

**Comment by devdesco-ceo** [comment]
Please fix it! I still get the sysem exception and it is breaking the search function completely!

**Comment by tmotyl** [comment]
I had similar issue, and it was caused by some 3rd party extension  which did a solr search in some observer before rendering to check if there are any results. 
Later in the rendering when solarium tried to load results, the register was already set (because of the previous search).
